### DISCUSSION TITLE
Proto descripers with bundles

### DISCUSSION THREAD

**Discussion by brion-fuller** [open-policy-agent/community#637] [Envoy]
Hello,
I started testing some cases I have with Envoy OPA plugin.  I have a setup in which works well with a single service and single opa instance following the describers [here](https://www.openpolicyagent.org/docs/latest/envoy-introduction/#configuration).  The problem is from the docs it looks like only a single proto-descriptor file is supported.  Is there a current way to support multiple proto descriptors?

**Comment by srenatus** [comment]
You should be able to use protoc to combine them. I don't know the command by heart but perhaps that's enough for you to find it. I'll check my notes later!

**Reply by brion-fuller** [reply]
Is there a way to bundle the proto describers along with the policies?

**Reply by srenatus** [reply]
There is not, no. Right now, they need to be provided out of band.
